apps/examples/slsiwifi/nettest_functions.c

```diff
@@ -105,7 +105,7 @@ static int ping_try_counter = 0;
 #define PING_RESULT(ping_ok)
 
 static u16_t ping_seq_num;
-static systime_t ping_time;
+static clock_t ping_time;
```

Comment:
ping_time has a return value of sys_now(). sys_now uses clock_systimer inside kernel space (lwip).
Is this needed?